test/integration/indexer/indexing_test.rb

```diff
@@ -57,6 +57,25 @@ def test_tagging_organisations_to_self
     })
   end
 
+  def test_parts_of_taxonomy
```

Comment:
Not sure about this integration test. Apps will never not send `part_of_taxonomy` directly via a POST.

Would it be better to skip this test and add something extra to the links lookup test?